packages/react-components/src/components/utils/ChatMessageComponentUtils.tsx

```diff
@@ -150,3 +150,19 @@ export const getMessageEditedDetails = (
   }
   return undefined;
 };
+
+/**
+ * Removes Fluent UI keyboard navigation styles from the given HTMLDivElement.
+ *
+ * This function removes the 'tabindex' and 'data-tabster' attributes from the provided
+ * HTMLDivElement, making it non-focusable and removing any associated keyboard navigation styles.
+ *
+ * @param node - The HTMLDivElement from which to remove the keyboard navigation styles. If null, the function does nothing.
+ */
+export const removeFluentUIKeyboardNavigationStyles = (node: HTMLDivElement | null): void => {
```

Comment:
Curious; how stable is this going to be?
Do we have tests to cover the changes when fluent gets updated?

Reply:
It's removed every time when a body component ref is set or changed (see the usage).
Yeah, I was thinking about the testing too and one of the cases that I see is to test that the keyboard navigation is actually correct but not sure what else we can do here.

Please share suggestions if any :)

Reply:
Added the keyboard navigation tests